scripts/clear-all-fault-leds.sh

```diff
@@ -4,6 +4,14 @@
 # /xyz/openbmc_project/State/Decorator/OperationalStatusManager
 # to true or false.
 
+# Skip running this script during a warm reboot(graceful/force).
+current_chassis_status=$(busctl get-property xyz.openbmc_project.State.Chassis /xyz/openbmc_project/state/chassis0 xyz.openbmc_project.State.Chassis CurrentPowerState | cut -d" " -f2)
```

Comment:
To cover possible issues with the script showing incomplete output, please add a -l to the busctl call.

Reply:
You can ignore this. Looks like `-l` only applies to the list sub-command.